refresh.py

```diff
@@ -659,8 +675,7 @@ def write_out_results(item: dict, item_name: str) -> None:
         print(f"Saving {main_folder} to {main_folder}/default.jsonl")
         os.makedirs(main_folder, exist_ok=True)
 
-        item.reset_index(inplace=True)
-        item.to_json(f"{main_folder}/default.jsonl", orient="records", lines=True)
+        item.reset_index(drop=True).to_json(f"{main_folder}/default.jsonl", orient="records", lines=True)
```

Comment:
This is probably the cause for removing a lot of examples. drop=True will remove items due to them having the same index (e.g. if you concat two data frames where both start their index at 1).

Reply:
Maybe. I tried to remove, Index_0 from result tables. I'll try to make this in `to_json`
![image](https://github.com/user-attachments/assets/1aa9b769-dc71-44f2-b806-394b901c9420)

Reply:
According to the [docs](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_json.html) it shouldn't add the index to the json when orient=record. So it might be that the column was accidentally added?

Reply:
When done reset_index it becomes additional column `Unnamed: 0` and exoprted to_json, so maybe we shouldn't reset_index before exoprt

Reply:
Ahh right. I believes it gives an error otherwise (but if not just remove it). Otherwise drop the column before writing it